lib/utils.py

```diff
@@ -376,3 +376,37 @@ def str_to_bytes(s):
     if PY2:
         return s
     return s.encode('utf-8')
+
+
+def parse_tag_line(line):
+    tags_str = line.split(':', 1)[1].strip()
+    return [tag.strip() for tag in tags_str.split(',')]
+
+
+def find_tags(filename):
+    """ Extract tags from a first comment in the file.
+    """
```

Comment:
This remark is not associated with your pull request, but with all docstrings through the project. In spite of citing PEP257 our docstrings have no identical format. I think we have to change this and fix the format for one-line and multi-line docstrings in our code style.